project/plugins.sbt

```diff
@@ -3,3 +3,5 @@ addSbtPlugin("com.github.gseitz" % "sbt-release" % "1.0.11")
 addSbtPlugin("com.jsuereth" % "sbt-pgp" % "1.1.2")
 
 addSbtPlugin("org.xerial.sbt" % "sbt-sonatype" % "3.8")
+
+addSbtPlugin("net.virtual-void" % "sbt-dependency-graph" % "0.10.0-RC1")
```

Comment:
As it's not required for this fix, could you remove this extra plugin dependency? The [doc](https://github.com/sbt/sbt-dependency-graph#usage-instructions) for `sbt-dependency-graph` suggests adding it to your global sbt configuration (`~/.sbt/1.0/plugins/plugins.sbt`) in the first instance, which I think is better - we shouldn't have to add this line to every single sbt project we work on!

Reply:
Oh I see! Thank you!

Reply:
agree,
worth noting that we do need it for any project that's checked by snyk as the snyk tool requires it. (this one is only checked transitively so I agree it's not needed in the codebase here)

Reply:
> we do need it for any project that's checked by snyk as the snyk tool requires it

Ooh, I didn't know that!